test_cluster_detection.py

```
@@ -0,0 +1,242 @@
+    and standard deviation 
+
+    PARAMETERS
+    ----------
+    n_samples: list[int]
+        list corresponding to how many points to generate for each generated cluster
+        ex: [10 20 30] will generate 10 points for one cluster, 20 points for the next, 
+        and 30 points for the final cluster
+
+    cluster_standard_deviation: int
+        the standard deviation of the generated points, bigger
+        standard deviation == more spread out points
+
+    RETURNS
+    -------
+    """
+    # .make_blobs() is a sklearn library function that returns a tuple of two values
+    # first value is ndarray of shape (2, total # of samples) that gives the (x,y) 
+    # coordinate of generated data points
+    # second value is the integer labels for cluster membership of each generated point.
+
+    generated_points, labels, cluster_positions = sklearn.datasets.make_blobs(n_samples=n_samples, 
+                                       n_features=2, 
+                                       cluster_std=cluster_standard_deviation, 
+                                       center_box=(0,CENTER_BOX_SIZE), 
+                                       random_state=0,
+                                       return_centers=True)
+    detections = []
+    for point in generated_points:
+        # Placeholder variables to create DetectionInWorld objects
+        placeholder_verticies = np.array([[0,0],[0,0],[0,0],[0,0]])
+        placeholder_label = 1
+        placeholder_confidence = 0.5
+
+        detection_created, detection_to_add = DetectionInWorld.create(placeholder_verticies,
+                                                                      point,
+                                                                      placeholder_label,
+                                                                      placeholder_confidence)
+        
+        if (detection_created):
+            detections.append(detection_to_add)
```

Just assert instead, we don't want to silence errors in unit tests.